ddtrace/contrib/openai/patch.py

```diff
@@ -392,23 +419,21 @@ def _handle_response(self, pin, span, integration, resp):
         """
 
         def shared_gen():
-            stream_span = pin.tracer.start_span("openai.stream", child_of=span, activate=True)
             try:
                 num_prompt_tokens = span.get_metric("openai.response.usage.prompt_tokens") or 0
                 num_completion_tokens = yield
 
-                stream_span.set_metric("openai.response.usage.completion_tokens", num_completion_tokens)
+                span.set_metric("openai.response.usage.completion_tokens", num_completion_tokens)
                 total_tokens = num_prompt_tokens + num_completion_tokens
-                stream_span.set_metric("openai.response.usage.total_tokens", total_tokens)
+                span.set_metric("openai.response.usage.total_tokens", total_tokens)
+                integration.metric(span, "dist", "tokens.prompt", num_prompt_tokens, tags=["openai.estimated:true"])
                 integration.metric(
                     span, "dist", "tokens.completion", num_completion_tokens, tags=["openai.estimated:true"]
                 )
                 integration.metric(span, "dist", "tokens.total", total_tokens, tags=["openai.estimated:true"])
             finally:
-                stream_span.finish()
```

Comment:
is this the same span from `_traced_endpoint()`? Or is this a separate stream span but just renamed?